server/utils/articles/base.util.js

```diff
@@ -35,7 +35,7 @@ class baseUtils {
   }
 
   static isBoolean(value) {
-    if (typeof value !== 'boolean') return false;
+    if (typeof value !== 'boolean' && value !== true && value !== false && value !== 'true' && value !== 'false') return false;
```

Comment:
Please i would like to seek clarity on this IF statement here.

Reply:
> Please i would like to seek clarity on this IF statement here.

So for you to test an image on postman, you need to use form-data which takes in input as strings but we validated published to be boolean so i had to cater for when it is not true or false and also had to coerce the truthy and falsy values so they can be used in the form-data.